scratch-arduino-extensions/scripts/patch-gui.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const fs = require('fs');
+
+const ExtIds = ["IncludeRobot", "Scratch3Arduino"];
+const ExtDirName = "scratch3_arduino";
+const ExtDirPath = path.resolve("./scratch-vm/src/extensions/", ExtDirName);
+
+const GuiDir = "../scratch-gui";
```

Comment:
```suggestion
const GuiDir = "./scratch-gui";
```

Reply:
This fix is not needed, when I suggested it, my folder structure was not following the right structure. Sorry for the noise.